lms/views/canvas/config.py

```diff
@@ -7,10 +7,74 @@
     request_method="GET",
 )
 def config_xml(request):
-    """Render the XML configuration."""
+    """
+    Render the XML configuration.
```

Comment:
```suggestion
    Generate XML configuration for the admin pages in Canvas.
    
    This allows an admin user to paste in a URL and have Hypothesis automatically
    configured in Canvas.
```

Maybe something like this? Also seeing as this extra detail applies to both, maybe this would good as a file level doc string?